README.md

```diff
@@ -47,6 +47,34 @@ To see live changes to the frontend while developing,
 * Run ```npm run build-watch```  
 * Hard refresh your browser
 
+# Python debugger
+
+To use [`ipdb`](https://github.com/gotcha/ipdb), a Python debugger, follow these steps:
+
+1. Open `docker-compose.yml`. Under the `web` key, insert `stdin_open: true` and `tty: true`. It should look like this:
+
+```yaml
+services:
+  web:
+    stdin_open: true
+    tty: true
+```
+
+2. Run `docker-compose up`.
+3. Run `docker ps`. Find the value in the "CONTAINER ID" column that corresponds to the `eahuborg_web` image.
+4. Using the ID you found in the previous step, run `docker attach <container ID>`. The command should be something like `docker attach 362618269c67`.
+5. Insert `import ipdb; ipdb.set_trace()` wherever you need a debugger statement.
```

Comment:
I personally prefer to setting up the env variable `PYTHONBREAKPOINT=ipdb.set_trace` and then just use `breakpoint()` in the code